configs/lip-a2/microscope.j5

```diff
@@ -1,35 +1,79 @@
+        "tsettle_motion_base": 0.3,
+        //2023-09-04: saw miss at 0.2
+        "tsettle_hdr": 0.3,
     },
     "motion": {
         "hal": "grbl-ser",
 
         "backlash": 0.05,
         "backlash_compensation": -1,
-        "z_format6": true,
+        "xyz_format6": true,
 
         "use_wcs_offsets": true,
+        /*
+        2023-09-06
+        X289 Y189 Z-65
+        1:20 to first hit z limit switch
+        2:35 to complete z homing
+        3:15: x axis hit (after y)
+        4:40: done
+        */
+        "max_home_time": 300,
         "grbl": {
             "rc_pre_home": [
                 //$100, $101 and $102 – [X,Y,Z] steps/mm
-                //Z is actually 5x this but we'll loose precision if here
-                //so instead apply that reduction host side
-                "$100=800.000", "$101=800.000", "$102=800.000",
+                //"$100=800.000", "$101=800.000", "$102=800.000",
+                //Eliminate rounding errors => 8/5 adjustment
+                //However we need to basically rescale everything now
+                "$100=500.000", "$101=500.000", "$102=500.000",
 
                 //2023-02-24: suspect missing steps. Make less aggressive
                 //X - Max Rate (mm/min)
                 //Default
                 //"$110=1200.000", "$111=1200.000", "$112=1200.000",
-                "$110=600.000", "$111=600.000", "$112=600.000",
+                //"$110=600.000", "$111=600.000", "$112=600.000",
+                //8/5 adjustment
+                "$110=960.000", "$111=960.000", "$112=1920.000",
                 //$120, $121, $122 – [X,Y,Z] Acceleration, mm/sec^2
                 //Default values, didn't try changing
                 //"$120=30.000", "$121=30.000", "$122=30.000",
-                "$120=15.000", "$121=15.000", "$122=15.000",
+                //8/5 adjustment
+                "$120=24.000", "$121=24.000", "$122=24.000",
 
+                //max travel
+                //"$130=300.000", "$131=200.000", "$132=60.000",
+                //"$130=480.000", "$131=320.000", "$132=96.000",
+                //z axis gearbox + 0.9 motor
+                "$130=480.000", "$131=320.000", "$132=960.000",
+
+                //Homing adjustments to account for gearbox etc
+                //"$24=25.0","$25=500.000",
+                //"$26=250","$27=15.000",
+                //8/5 adjustment
+                //$24: Homing feed, mm/min
+                //$25: Homing seek, mm/min
+                "$24=40.0","$25=800.000",
+                //$26: Homing debounce, milliseconds
+                "$26=400",
+                //$27: Homing pull-off, mm
+                //x/y: 24 * 5/8 => 15 mm
+                //z: 24 / (8/5 * 10) => 1.5 mm
+                //z:  48 => 3.0 mm
+                //2023-10-12: had increased this to be within soft limits, but it makes it take a really long time to home
+                "$27=5.000",

```

Comment:
this is probably sub-optimal w/o gearbox (homing will take much longer than it needs to), but should be fine for now

Reply:
👍🏼